SimGeneral/MixingModule/python/mixNoPU_PPS_cfi.py

```diff
@@ -0,0 +1,58 @@
+import FWCore.ParameterSet.Config as cms
+
+# this is a minimum configuration of the Mixing module,
+# to run it in the zero-pileup mode
+#
```

Comment:
Why is this file needed?

Reply:
no, I mean the "mixNoPU_cfi.py" because the former developers created the "mixNoPU_PPS_cfi.py"

Reply:
Yes, you are right. I checked it again (with the developers). The reason was that to run a test with a particle gun, there was an error that was solved by the dedicated mixNoPU_PPS_cfi. I'm trying to solve the problem due to missing calorimeter geometry which might be caused by missing calorimeter objects in a simple particle gun run.